actors.md

```diff
@@ -1022,19 +1022,177 @@ func ChangeWorker(addr Address) {
 }
 ```
 
-### Payment Channel Broker Actor
+### Payment Channel Actor
+
+- **Code Cid:** `<codec:raw><mhType:identity><"paych">`
+
+The payment channel actor manages the on-chain state of a point to point payment channel.
+
+```go
```

Comment:
- downcase the field names
- `UInt` (capitial `I`)
- lets use `sh` for highlighting to fit with the other code blocks